devicemodel/core/main.c

```diff
@@ -171,7 +172,8 @@ usage(int code)
 		"       --vtpm2: Virtual TPM2 args: sock_path=$PATH_OF_SWTPM_SOCKET\n"
 		"       --lapic_pt: enable local apic passthrough\n"
 		"       --rtvm: indicate that the guest is rtvm\n"
-		"       --logger_setting: params like console,level=4;kmsg,level=3\n",
+		"       --logger_setting: params like console,level=4;kmsg,level=3\n"
+		"       --pm_notify_channel: define the channel used to notify guest about power event\n",
```

Comment:
Can you also update `doc/user-guides/acrn-dm-parameters.rst` please?

Reply:
Geoff, sure. The document update is on our todo list. There will be one more patchset will add new parameter of DM. Is it OK that we update the dm parameters document after that patchset is merged?

Reply:
I'd prefer to do this directly in this patch (given it's such a trivial doc change) but OK to do it later.

Reply:
OK. Will update the document when I create new PR.

Reply:
Thanks!